README.md

```diff
@@ -4,54 +4,48 @@ Shell script alias manager.
 
 ## Getting Started
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. 
 
 ### Prerequisites
 
-What things you need to install the software and how to install them
-
-```
-Give examples
-```
+Before starting with Ali you must have installed Go.
 
 ### Installing
 
-A step by step series of examples that tell you how to get a development env running
-
-Say what the step will be
+Clone the repository in your computer:
 
 ```
-Give the example
+$ git clone git@github.com:adriacidre/ali.git
 ```
 
-And repeat
-
-```
-until finished
-```
-
-End with an example of getting some data out of the system or using it for a little demo
-
 ## Running the tests
 
-Explain how to run the automated tests for this system
+TBD
 
-### Break down into end to end tests
 
-Explain what these tests test and why
+### And coding style tests
+
+TBD
 
 ```
 Give an example
 ```
 
-### And coding style tests
+## How to use it
 
-Explain what these tests test and why
+To see a list of aliases:
 
 ```
-Give an example
+$ ./ali l
 ```
 
+To create a new aliases:
+
+```
+$ ./ali a
```

Comment:
this doesn't work, you must provide the alias name you want to setup like
```
$ ./ali a <myalias>
```

Reply:
You're right. Changes nave been made.